tests/sql.py

```diff
@@ -172,7 +177,7 @@ def setUp(self):
 
     def test_lastrowid(self):
         self.db.execute("CREATE TABLE foo(id INTEGER PRIMARY KEY AUTOINCREMENT, firstname TEXT, lastname TEXT)")
-        self.assertEqual(self.db.execute("INSERT INTO foo (firstname, lastname) VALUES('firstname', 'lastname')"), 1)
+        self.assertEqual(sv elf.db.execute("INSERT INTO foo (firstname, lastname) VALUES('firstname', 'lastname')"), 1)
```

Comment:
Afraid this isn't syntactically valid!

Reply:
It supposed to commit changes only at line 163. I will check where it went wrong, why one line is deleted and is changed by the same line.